web/index.html

```diff
@@ -95,8 +95,8 @@
         </div>
         <div class="view" flex two layout vertical>
           <div class="header" layout horizontal wrap>
-            <li><a id="consoletab" selected><span>Console</span></a></li>
-            <li><a id="doctab"><span>Documentation</span></a></li>
+            <li><a id="consoletab" title="CTRL+ENTER / ⌘+ENTER" selected><span>Console</span></a></li>
```

Comment:
I think we're ok with no title on this tab - it switches to this tab as a side-effect of running the app.